tensorflow_datasets/core/download/download_manager.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, url, tmp_path):
     Exception.__init__(self, msg)
 
 
-class DownloadConfig(object):
```

Comment:
We still want to support Python 2 internally, could you revert those changes ?

Reply:
Done